ProcessesApi/V1/Factories/CreateRequestFactory.cs

```diff
@@ -0,0 +1,38 @@
+using ProcessesApi.V1.Domain;
+using ProcessesApi.V1.Infrastructure;
+using ProcessesApi.V1.Boundary.Request;
+using System;
+using System.Collections.Generic;
+
+namespace ProcessesApi.V1.Factories
+{
+    public static class CreateRequestFactory
```

Comment:
Naming seems a bit off, it's not creating a request but a db entity object.

Reply:
True, but the other APIs (e.g. Person, Tenure) use the same name. I could put it in the `EntityFactory` class?